modules/app/managed-environment/main.bicep

```diff
@@ -17,13 +17,6 @@ param tags object?
 @description('Optional. Array of role assignments to create.')
 param roleAssignments roleAssignmentType
 
-@allowed([
-  'Consumption'
-  'Premium'
-])
-@description('Optional. Managed environment SKU.')
-param skuName string = 'Consumption'
```

Comment:
How come the SKU was removed? Did this parameter become rundant?

Reply:
Hi, yes the SKU name has been removed from the latest API
https://learn.microsoft.com/en-us/azure/templates/microsoft.app/change-log/managedenvironments